great_expectations/datasource/fluent/spark_file_path_datasource.py

```diff
@@ -328,8 +333,12 @@ def _get_reader_options_include(self) -> set[str]:
         return super()._get_reader_options_include().union({"merge_schema"})
 
 
-class DirectoryORCAsset(_DirectoryDataAssetMixin, ORCAsset):
-    type: Literal["directory_orc"] = "directory_orc"  # type: ignore[assignment]
+class ORCAsset(_RegexDataAssetMixin, ORCAssetBase):
+    type: Literal["orc"] = "orc"
+
+
+class DirectoryORCAsset(_DirectoryDataAssetMixin, ORCAssetBase):
+    type: Literal["directory_orc"] = "directory_orc"
```

Comment:
non blocking, but I feel like we should consider not following the convention of `<X>Asset` and `Directory<X>Asset`, but instead do like `File<X>Asset` and `Directory<X>Asset`